homeassistant/components/econet/water_heater.py

```diff
@@ -238,3 +248,23 @@ def min_temp(self):
     def max_temp(self):
         """Return the maximum temperature."""
         return self.water_heater.max_set_point
+
+    @property
```

Comment:
Why do you use the values twice? Once as an attribute and here as a property.

Reply:
Yes, you're correct. I don't need the properties. I removed them and retested locally and everything still works. I'll remove them and submit a new PR. Thanks.